sdk/resourcemanager/datafactory/armdatafactory/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 2.0.0 (2022-06-22)
+### Breaking Changes
+
+- Type of `LinkedServiceReference.Type` has been changed from `*LinkedServiceReferenceType` to `*Type`
```

Comment:
Are those breaking change expected and get approved before?

Reply:
test pr